python/lsst/dm/csc/base/dm_csc.py

```diff
@@ -59,6 +60,22 @@ async def configure(self, config):
 
         )
 
+    # this assumes a file name of the type:
+    # "CC_O_20200407_000004-R22S00.fits"
```

Comment:
There can be a problem in this code if they ask you to ingest CCS images as well. His naming convention is not the same as ours.

Reply:
There are several ways to determine the info from the file, none of them is great.   If we weren't concerned with speed, we'd open the file and read the information out of it.  But unfortunately for our proposes, it would take too long.   

When I asked about parsing the file name, I got this response:  
'The filename should be guaranteed to contain: "the image name, including the observation day (date in the UTC-12 timezone) and the image sequence number, as well as the raft and sensor of the image."
So while it's non-ideal to parse it out of the filename, it's safe.'

The format of the filename might change, in which case we'll have to revisit this code.  For now, it'll have to do.